Documentation/ExtensionInstallation/Index.rst

```diff
@@ -1,61 +1,85 @@
 .. include:: ../Includes.txt
 
-
 .. _extension-installation:
 
 ======================
-Extension Installation
+Extension installation
 ======================
 
 This tutorial assumes that your TYPO3 instance is a brand new installation,
 without any themes, templates, pages or content. See the :ref:`TYPO3
 Installation Guide <t3install:start>` for a detailed explanation how to set up
-a TYPO3 instance from scratch. For the sake of simplicity, it is also assumed
-that TYPO3 has been installed the *traditional way*, by extracting the source
-package into the web directory **without** using PHP *composer*.
+a TYPO3 instance from scratch.
 
-By using this method, extensions (e.g. the sitepackage extension) can be
-installed using the Extension Manager, which is a *module* found in the
-backend of TYPO3. Before we can install the sitepackage extension, we have to
-transfer the files from our local machine to the TYPO3 server (if all files and
-directories have been created on the local machine though).
+We highly recommend to use :ref:`the Composer -based installation process
+<t3install:composer-working-with>`. During development you should work locally
+on your machine for example by running TYPO3 on ddev.
 
-In case you have SSH/FTP access to the server, copy the directory
-:file:`site_package` (including all files and sub-directories) to the
-following directory in your TYPO3 instance: :file:`typo3conf/ext/`.
+If you need to follow the legacy way of installation, see :ref:`Site package
+installation without Composer <extension-installation_without_composer>`.
 
-If you do not have SSH/FTP access, create a ZIP file of the **content** of your
-:file:`site_package` folder. It is important that the ZIP archive does not
-contain the directory :file:`site_package` and its files and directories inside
-this folder. The files and folders must be directly located on the first level
-of ZIP archive.
 
-.. @TODO clarify, if the file name of the ZIP is relevant.
+.. _extension-installation_with_composer:
 
+Extension installation in Composer mode
+=======================================
 
-.. _typo3-backend-extension-manager:
+Starting with TYPO3 11.4 all extensions, including the site package extension
+must be installed via Composer .
```

Comment:
This should be clarified since it is only true if you chose composer mode.
Suggestion:
> Starting with TYPO3 11.4 if composer is used, all extensions, including the site package extension
must be installed via Composer.

Reply:
Well the headline just above is "Extension installation in Composer mode"

Reply:
Anyway to refuse confusion lets use your sentence. Can't do any harm...

Reply:
Yes, you are right :) I've read it a bit out of context.